src/activity-logger/background/pause-logging.js

```diff
@@ -60,16 +60,16 @@ function handlePause(timeout) {
         return timeout
     }
 
-    analytics.trackEvent({
-        category: 'Popup',
-        action: 'Resume indexing',
-    })
+    return setTimeout(() => {
+        analytics.trackEvent({
```

Comment:
I know this is not entirely related to this PR's purpose, but now is a good time to discuss:
This code gets called after the pause timer ends (so if I pause for 10 mins, this code will run after 10 mins). Given that, I don't really think the "resume indexing" event should be triggered here as it's not a directly user-invoked action. I think it would make much more sense in the `handleInterrupt` function (fires when the user chooses to manually unpause). What do you think about that?

Also: as we are tracking "resume indexing" event, shouldn't we also track when the user pauses indexing? (any opinion @oliversauter?)

Reply:
As we are not taking any data from the user perspective, so we have to see that when the user has indexing on, in future we can calculate the total time that user has enabled the indexing, then this event will be useful.
We are also tracking the pauseIndexing from the popup container.
Does that make sense?

Reply:
> As we are not taking any data from the user perspective, so we have to see that when the user has indexing on, in future we can calculate the total time that user has enabled the indexing, then this event will be useful.

TBH I don't entirely understand what you mean here. 
What I'm saying is: why not move this `trackEvent` call to where the user actually resumes indexing, as the event is meant to represent when the user chooses to "resume indexing"? (so `handleInterrupt`, not this function)
The time that the user pauses indexing for should be available as well; no need to calculate it.

Reply:
> We are also tracking the pauseIndexing from the popup container.

If we are tracking pausing indexing there, why not also track resuming indexing there?

Reply:
> If we are tracking pausing indexing there, why not also track resuming indexing there?

Suppose I resumed for 20 minuets, we will send the event that user has paused indexing for 20 mins. Now we don't know on which time the indexing starts again, as we are not takinng duration of pause.

Reply:
We can change the name of the event, `automaticResumeIndexing`.